.pre-commit-config.yaml

```diff
@@ -35,4 +35,5 @@ repos:
     hooks:
     -   id: check-hooks-apply
     -   id: check-useless-excludes
-python_version: python3.6
+default_language_version:
+    python: python3.6
```

Comment:
We should use 3.7

Reply:
I would prefer to stay on Python 3.6, as Bugbug targets Python 3.6, we know that all developers have at least Python 3.6 installed when working on bubug.

Reply:
I guess we should switch to targeting 3.7, as I'm already using some features that were added in 3.7.